### PR TITLE
Update TPC-DS Q90

### PR DESCRIPTION
**Pull request by sgrebnov**
## 🗣 Description

Update TPC-DS Q90. As explained in #2708 `cast(amc as decimal(15,4))/cast(pmc as decimal(15,4)) am_pm_ratio` can't be performed on test data we have and result in `ERROR: division by zero` and similar errors. Updating the filter so the query can be performed.

## 🔨 Related Issues

https://github.com/spiceai/spiceai/issues/2708
https://github.com/spiceai/spiceai/issues/2960

### CODE REVIEW

File: crates/runtime/benches/queries/tpcds/q35.sql [hunk 1]
```diff
@@ -10,8 +10,8 @@ select
   cd_dep_employed_count,
   count(*) cnt2,
   max(cd_dep_employed_count),
-  stddev_samp(cd_dep_employed_count),
```
Comment (sgrebnov) [inline crates/runtime/benches/queries/tpcds/q35.sql:13]: This is unrelated change, additional update for https://github.com/spiceai/spiceai/pull/3069

**Review by peasee — APPROVED** [review]
